fcrepo-kernel-impl/src/test/java/org/fcrepo/kernel/impl/FedoraResourceImplTest.java

```diff
@@ -395,6 +398,14 @@ public void testGetProperty() throws RepositoryException {
     }
 
     @Test
+    public void testSetURIProperty() throws URISyntaxException, RepositoryException {
+        final String prop = "premis:hasEventRelatedObject";
+        final String uri = "http://localhost:8080/rest/1";
```

Comment:
I also wonder about passing in values as full URLs that do not pass through an idTranslator. Is this a concern?